CRM/Civicase/Service/CaseMenu/DashboardRoute.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+use CRM_Civicase_Helper_CaseCategory as CaseCategoryHelper;
+
+/**
+ * Class for Dashboard route.
+ */
+class CRM_Civicase_Service_CaseMenu_DashboardRoute extends CRM_Civicase_Service_CaseMenu_Route {
+
+  /**
+   * {@inheritDoc}
+   */
+  protected function label() {
+    return ts('Dashboard');
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  protected function name() {
+    return "{$this->categoryName}_dashboard";
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  protected function url() {
+    $url = '';
+    $url .= 'civicrm/case/a/';
+
+    if ($this->categoryName !== CaseCategoryHelper::CASE_TYPE_CATEGORY_NAME) {
+      $url .= "?case_type_category={$this->categoryId}";
+      $url .= "&p=dd";
```

Comment:
This line can be put outside of if-else, as its common.

Reply:
Right, fixed: https://github.com/compucorp/uk.co.compucorp.civicase/pull/756/commits/dfec6d26954ba68d4e193996c85b191f77596e1a